winspool/winspool.go

```diff
@@ -154,7 +154,7 @@ func convertPrinterState(wsStatus uint32) *cdd.PrinterStateSection {
 		}
 		state.VendorState.Item = append(state.VendorState.Item, vs)
 	}
-	if wsStatus&PRINTER_STATUS_OFFLINE != 0 {
+	if wsStatus&PRINTER_STATUS_OFFLINE != 0 || wsAttributes&PRINTER_ATTRIBUTE_WORK_OFFLINE != 0 {
```

Comment:
I investigated this a bit, it looks like we probably don't want to change how we report Work Offline printers. Specifically, this looks like a Windows feature that lets a user explicitly hide the fact that a printer is disconnected. I don't think we want to try to detect this and undo this intention.